### PR TITLE
Remove the blue sphere while speaking

### PR DESCRIPTION
**Pull request by ZappoMan**
Remove this feature at Philip's request.

### CODE REVIEW

**Comment by ZappoMan** [comment]
Note: This PR is based off of the Advanced/Developers menus PR, so please merge that one first.

File: interface/src/Menu.h [hunk 1]
```diff
@@ -151,7 +168,6 @@ namespace MenuOption {
     const QString Back = "Back";
     const QString BandwidthDetails = "Bandwidth Details";
     const QString BinaryEyelidControl = "Binary Eyelid Control";
-    const QString BlueSpeechSphere = "Blue Sphere While Speaking";
```
Comment (ZappoMan) [inline interface/src/Menu.h]: This change is specific to removing this feature.

File: interface/src/avatar/Avatar.cpp [hunk 2]
```diff
@@ -495,39 +495,6 @@ void Avatar::render(RenderArgs* renderArgs, const glm::vec3& cameraPosition) {
                 }
             }
         }
-
-        // quick check before falling into the code below:
-        // (a 10 degree breadth of an almost 2 meter avatar kicks in at about 12m)
-        const float MIN_VOICE_SPHERE_DISTANCE = 12.0f;
-        if (Menu::getInstance()->isOptionChecked(MenuOption::BlueSpeechSphere)
```
Comment (ZappoMan) [inline interface/src/avatar/Avatar.cpp]: This change is specific to removing this feature.

**Comment by hifi-gustavo** [comment]
Build console output for commit: be058398a2477cfe7ee0c7a67faccc47aaaeeb14 available at: https://gist.github.com/84aedd9b8c59ec92a868<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/High+Fidelity-PR-6641-win64.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6641.dmg

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: be058398a2477cfe7ee0c7a67faccc47aaaeeb14 available at: https://gist.github.com/84aedd9b8c59ec92a868<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/High+Fidelity-PR-6641-win64.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6641.dmg

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: be058398a2477cfe7ee0c7a67faccc47aaaeeb14 available at: https://gist.github.com/84aedd9b8c59ec92a868<br/><br/>No installer was built for this commit

**Comment by hifi-gustavo** [comment]
Build console output for commit: 248c94e6389da7dfcf36dfa8cebc0841fb10c0d8 available at: https://gist.github.com/84aedd9b8c59ec92a868<br/><br/>No installer was built for this commit

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 248c94e6389da7dfcf36dfa8cebc0841fb10c0d8 available at: https://gist.github.com/84aedd9b8c59ec92a868<br/><br/>No installer was built for this commit

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 248c94e6389da7dfcf36dfa8cebc0841fb10c0d8 available at: https://gist.github.com/84aedd9b8c59ec92a868<br/><br/>No installer was built for this commit

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 248c94e6389da7dfcf36dfa8cebc0841fb10c0d8 available at: https://gist.github.com/84aedd9b8c59ec92a868<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/High+Fidelity-PR-6641-win64.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6641.dmg

**Comment by Atlante45** [comment]
@leo Can I merge this or are you still testing?

**Comment by murillodigital** [comment]
@Atlante45 good to merge

**Comment by leo** [comment]
@Atlante45 I think you got the username wrong :see_no_evil:

**Comment by murillodigital** [comment]
@leo sorry about that, I think this is not the first time you get stuff meant for me :P

**Comment by Atlante45** [comment]
@leo Damn, yes my bad. Slack habit.

**Comment by leo** [comment]
@murillodigital @Atlante45 No problem, guys :blush:

**Pull request merged**